libindi/drivers/telescope/telescope_script.cpp

```diff
@@ -79,7 +78,11 @@ const char * ScopeScript::getDefaultName() {
 bool ScopeScript::initProperties() {
   INDI::Telescope::initProperties();
   
+#ifdef OSX_EMBEDED_MODE
   IUFillText(&ScriptsT[0], "FOLDER", "Folder", "/usr/local/share/indi/scripts");
+#else
+  IUFillText(&ScriptsT[0], "FOLDER", "Folder", "/usr/share/indi/scripts");
```

Comment:
I have an issue with sticking things in /usr and not in /usr/local. I know that @knro prefers the first but I don't know why. It is against Debian guidelines (?), and it messes up with package management. Jasem, would you care to comment?